pkg/drainer/drainer.go

```diff
@@ -28,50 +28,59 @@ import (
 
 var drainHelper *drain.Helper
 var node *corev1.Node
+var nthConfig config.Config
 
-//InitDrainer will ensure the drainer has all the resources necessary to complete a drain
-func InitDrainer(nthConfig config.Config) {
+// InitDrainer will ensure the drainer has all the resources necessary to complete a drain.
+// This function must be called before any function in the drainer package.
+func InitDrainer(inputNthConfig config.Config) {
+	nthConfig = inputNthConfig
```

Comment:
It's particularly odd seeing this here, where you're copy-constructing a global-scoped variable to the value of a parameter that you pass to the outer function...

Reply:
yep, agreed! refactored drainer file